src/WalkingModule/app/robots/iCubGenova04/dcm_walking/joypad_control/plannerParams.ini

```diff
@@ -1,5 +1,5 @@
 ##Timings
```

Comment:
Please avoid changing this file since we are using this gains for public demos. If you think there is a problem please open an issue and an associate PR only for this

Reply:
solved!